clients/imodels-client-management/src/base/internal/IModelsErrorParser.ts

```diff
@@ -112,9 +116,20 @@ export class IModelsErrorParser {
     return result;
   }
 
-  private static resolveUnauthorizedMessage(responseBody?: unknown): string {
-    return (responseBody as IModelsApiErrorWrapper)?.error?.message
-      ?? (responseBody as UnwrappedError)?.message
+  private static createUnrecognizedError(response: ResponseInfo): Error {
+    return new IModelsErrorImpl({
+      code: IModelsErrorCode.Unrecognized,
+      message: `${IModelsErrorParser._defaultErrorMessage}. Response status code: ${response.statusCode}, response body: ${JSON.stringify(response.body)}`
```

Comment:
You are throwing this error when response does not have a body

Reply:
Changed the error message to include additional properties apart from response body/status code:
https://github.com/iTwin/imodels-clients/pull/160/commits/68afc00d7abcab50c6b762b21b521cd7d95f81b9
Now the resulting error message looks like this:
![image](https://user-images.githubusercontent.com/70565417/234843800-999a8f18-b124-4d69-98bc-1d906cacdaae.png)
the error is readable even if body is not returned